driver/bpf/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ if(BUILD_BPF)
 	set(bpf_min_kver_map_x86_64 4.14)
 	set(bpf_min_kver_map_aarch64 4.17)
 	set(bpf_min_kver_map_s390x 5.5)
+	set(bpf_min_kver_map_ppc64le 4.18)
```

Comment:
I am pretty sure that eBPF support will also need same fixes as modern eBPF, ie: 
* https://github.com/falcosecurity/libs/blob/master/driver/bpf/plumbing_helpers.h#L125
* https://github.com/falcosecurity/libs/blob/master/driver/bpf/plumbing_helpers.h#L167
* https://github.com/falcosecurity/libs/blob/master/driver/bpf/plumbing_helpers.h#L286

I also noticed that we skipped an include here: https://github.com/falcosecurity/libs/blob/master/userspace/libscap/engine/gvisor/parsers.cpp#L35.
I am not sure whether it is needed, but i guess so given we added aarch64 and s390x too (btw gvisor should only be supported on x86_64 atm, since the option is only available on it: https://github.com/falcosecurity/libs/blob/master/cmake/modules/engine_config.cmake#L26)

Reply:
Okay. I was able to build bpf and kmod out of the box without any changes, so I assumed I did not need to make any other modifications. I will add the fixes you suggested.

Reply:
@FedeDP Btw, I noticed these 2 lines point to incorrect links for each architecture.
https://github.com/falcosecurity/libs/blob/f61e7fff24f951af1bbe033416f09fdd439ab6b0/driver/bpf/plumbing_helpers.h#L268

https://github.com/falcosecurity/libs/blob/f61e7fff24f951af1bbe033416f09fdd439ab6b0/driver/bpf/plumbing_helpers.h#L289

Reply:
I think `libbpf` touched those files (we should've linked a fixed hash instead of master). No problem anyway!